modules/languages/ruby.nix

```diff
@@ -0,0 +1,83 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib;
+with builtins; let
+  #format-env = pkgs.callPackage ./format-derivation.nix {inherit pkgs;};
+  cfg = config.vim.languages.ruby;
+
+  defaultServer = "rubocop";
+  servers = {
+    solargraph = {
+      package = pkgs.ruby_3_2;
+
+      lspConfig = ''
+        lspconfig.solargraph.setup {
+          on_attach = attach_keymaps,
+          cmd = { "${cfg.lsp.package}/bin/bundle", "exec", "solargraph", "stdio" }
+        }
+      '';
+    };
+
+    rubocop = {
+      package = pkgs.ruby_3_2;
+
+      lspConfig = ''
+        lspconfig.rubocop.setup {
+          on_attach = attach_keymaps,
+          cmd = { "rubocop", "--lsp" },
```

Comment:
preferably the command should be `${pkgs.rubocop}/bin/rubocop`

Reply:
I don't use debuggers but ok will do